src/modules/core/components/Popover/PopoverSection/MetaSection.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { defineMessages } from 'react-intl';
+import { ActionButton } from '~core/Button';
+import { DropdownMenuItem, DropdownMenuSection } from '~core/DropdownMenu';
+import { ActionTypes } from '~redux/actionTypes';
+
+const MSG = defineMessages({
+  signOut: {
+    id: 'users.AvatarDropdown.AvatarDropdownPopover.link.signOut',
+    defaultMessage: 'Sign Out',
+  },
+});
+
+const displayName = 'users.AvatarDropdown.AvatarDropdownPopover.MetaSection';
+
+const MetaSection = ({ itemClassName }: { itemClassName?: string }) => (
```

Comment:
This is not a core component, it cannot be shared with others. It's a `user` level component